sdk/eventgrid/Azure.Messaging.EventGrid/api/Azure.Messaging.EventGrid.netstandard2.0.cs

```diff
@@ -32,29 +32,1185 @@ public partial class SharedAccessSignatureCredential
         public System.DateTimeOffset? Time { get { throw null; } set { } }
-        public string Type { get { throw null; } }
+        public string Type { get { throw null; } set { } }
+        public System.Collections.Generic.ICollection<object> Values { get { throw null; } }
+        public void Add(string key, object value) { }
+        public bool ContainsKey(string key) { throw null; }
+        public System.Collections.Generic.IEnumerator<System.Collections.Generic.KeyValuePair<string, object>> GetEnumerator() { throw null; }
+        public bool Remove(string key) { throw null; }
+        void System.Collections.Generic.ICollection<System.Collections.Generic.KeyValuePair<System.String,System.Object>>.Add(System.Collections.Generic.KeyValuePair<string, object> value) { }
+        void System.Collections.Generic.ICollection<System.Collections.Generic.KeyValuePair<System.String,System.Object>>.Clear() { }
+        bool System.Collections.Generic.ICollection<System.Collections.Generic.KeyValuePair<System.String,System.Object>>.Contains(System.Collections.Generic.KeyValuePair<string, object> value) { throw null; }
+        void System.Collections.Generic.ICollection<System.Collections.Generic.KeyValuePair<System.String,System.Object>>.CopyTo(System.Collections.Generic.KeyValuePair<string, object>[] destination, int offset) { }
+        bool System.Collections.Generic.ICollection<System.Collections.Generic.KeyValuePair<System.String,System.Object>>.Remove(System.Collections.Generic.KeyValuePair<string, object> value) { throw null; }
+        System.Collections.IEnumerator System.Collections.IEnumerable.GetEnumerator() { throw null; }
+        public bool TryGetValue(string key, out object value) { throw null; }
     }
     public partial class EventGridEvent
     {
         public EventGridEvent(string id, string subject, object data, string eventType, System.DateTimeOffset eventTime, string dataVersion) { }
-        public object Data { get { throw null; } }
-        public string DataVersion { get { throw null; } }
-        public System.DateTimeOffset EventTime { get { throw null; } }
-        public string EventType { get { throw null; } }
-        public string Id { get { throw null; } }
+        public object Data { get { throw null; } set { } }
+        public string DataVersion { get { throw null; } set { } }
+        public System.DateTimeOffset EventTime { get { throw null; } set { } }
+        public string EventType { get { throw null; } set { } }
+        public string Id { get { throw null; } set { } }
         public string MetadataVersion { get { throw null; } }
-        public string Subject { get { throw null; } }
+        public string Subject { get { throw null; } set { } }
         public string Topic { get { throw null; } set { } }
     }
 }
+namespace Azure.Messaging.EventGrid.Models.SystemEvents
```

Comment:
I thought we were going with `Azure.Messaging.EventGrid.SystemEvents`

Reply:
My bad - I don't know when it switched back